addons/script-collapse/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Collapsible scripts",
+  "description": "Enables script collapsing in the editor. ",
+  "tags": ["editor", "beta"],
+  "info": [
+    {
+      "type": "notice",
+      "text": "Other users will not be able to see collapsed scripts.",
```

Comment:
ofc? remove the notice

Reply:
Maybe the addon could add a comment like "collapsed" to remember collapsed scripts.

Reply:
What do you mean?

Reply:
> Maybe it's actually worth noting that if you reload the project, all the scripts will be uncollapsed, even if they were collapsed before.

Yeah, that is probably a good idea...even if you just hit `see project page`. I am thinking about using localstorage?

Reply:
What do you mean?

Reply:
🙄 That would require a TON of work.

Reply:
Yeah.